src/core/ext/xds/xds_client.cc

```diff
@@ -1471,7 +1472,8 @@ XdsClient::XdsClient(
       transport_factory_(std::move(transport_factory)),
       request_timeout_(resource_request_timeout),
       xds_federation_enabled_(XdsFederationEnabled()),
-      api_(this, &grpc_xds_client_trace, bootstrap_->node(), &symtab_),
+      api_(this, &grpc_xds_client_trace, bootstrap_->node(), &symtab_,
+           user_agent_name, user_agent_version),
```

Comment:
Please use `std::move()` for these two parameters to avoid unnecessary string copies.

Reply:
Done